packages/client/src/utils/validate.ts

```diff
@@ -743,6 +750,32 @@ export const isInformantOfLegalAge: Validation = (value: IFormFieldValue) => {
   }
 }
 
+export const isAgeInYearsBetween: Validation = (age: IFormFieldValue) => {
+  // Ensure the value is a number
+  const parsedAge = typeof age === 'number' ? age : Number(age)
+
+  // Validate the parsed value
+  if (isNaN(parsedAge)) {
+    return {
+      message: messages.invalidAge // Add a specific message for invalid values
+    }
+  }
+
+  // Access minimum and maximum age from the configuration
+  const minAge = window.config.INFORMANT_MINIMUM_AGE
+  const maxAge = window.config.INFORMANT_MAXIMUM_AGE
```

Comment:
We do not want to increase the number of configuration properties, we want to remove these properties entirely.